tests/unit_tests/models/test_deepchem_model.py

```diff
@@ -0,0 +1,41 @@
+from unittest import TestCase
+
+from deepchem.models import GraphConvModel
+
+from deepmol.compound_featurization import ConvMolFeat
+from deepmol.metrics.metrics_functions import roc_auc_score, precision_score, accuracy_score, confusion_matrix, \
+    classification_report
+
+from deepmol.metrics import Metric
+from deepmol.models import DeepChemModel
+from deepmol.splitters import SingletaskStratifiedSplitter
+from unit_tests.models.test_models import ModelsTestCase
+
+
+class TestDeepChemModel(ModelsTestCase, TestCase):
+
+    def test_fit_predict_evaluate(self):
+        ds = ConvMolFeat().featurize(self.mini_dataset_to_test)
+        splitter = SingletaskStratifiedSplitter()
```

Comment:
I believe this is an integration test, as it tests different components (1 featurizer, 1 splitter and the model) and not only the model. Nevertheless, I don't think it is critical, we would've to make mocks for the featurizer and splitter. I can open an issue with this.